### PR TITLE
Create .Net5 C#

### PR DESCRIPTION
**Pull request by jattanjie21**

### CODE REVIEW

**Comment by shiftkey** [comment]
@jatta-njie21 I currently cannot merge this as the project is in the incorrect location on disk - please follow the instructions in the previous comment to get the PR updated...

**Comment by jattanjie21** [comment]
Thank you @shiftkey checking the comments right away.

**Comment by jattanjie21** [comment]
Oh my bad, I just fixed the issue.

**Comment by shiftkey** [comment]
@jatta-njie21 the PR automation checks several things, and this last stage is just suggestions for common tags to avoid duplicating on similar terms

**Comment by jattanjie21** [comment]
> @jatta-njie21 the PR automation checks several things, and this last stage is just suggestions for common tags to avoid duplicating on similar terms

Noted. Updated the necessary changes.

File: _data/projects/.NetCSharp.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/jatta-njie21/CSharp-Practice
+tags:
+  - .net-core
+  - first-timers-only
+  - c-sharp
+  - .net
+  - beginner-friendly
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/username/project/labels/up%20for%20grabs
```
Comment (shiftkey) [inline _data/projects/.NetCSharp.yml]: I think this is the change that's needed

```suggestion
  link: https://github.com/jatta-njie21/CSharp-Practice/labels/up%20for%20grabs
```

File: _data/projects/.NetCSharp.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/jatta-njie21/CSharp-Practice
+tags:
+  - .net-core
+  - first-timers-only
+  - c-sharp
+  - .net
+  - beginner-friendly
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/jatta-njie21/project/labels/up%20for%20grabs
```
Comment (shiftkey) [inline _data/projects/.NetCSharp.yml]:
```suggestion
  link: https://github.com/jatta-njie21/CSharp-Practice/labels/up%20for%20grabs
```

Comment (ritwik12) [reply _data/projects/.NetCSharp.yml]: @jatta-njie21 reminder

**Comment by ghost** [comment]
@jattanjie21 are you still trying to add your repository? You seem to have forgotten about this PR for a few months.

**Comment by shiftkey** [comment]
Closing this out due to inactivity. 

@jattanjie21 if you're still interested in adding the project please submit a new pull request and consider the feedback from this discussion.

**Pull request closed without merging**